assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -9,38 +9,48 @@ interface State {
 
 interface Context {
 	woocommerce: {
-		productGallery: { numberOfThumbnails: number };
+		selectedImage: string;
+		imageId: string;
 	};
 }
 
 interface Selectors {
 	woocommerce: {
-		productGallery: {
-			numberOfPages: ( store: unknown ) => number;
-		};
+		isSelected: ( store: unknown ) => boolean;
+	};
+}
+
+interface Actions {
+	woocommerce: {
+		handleClick: ( context: Context ) => void;
```

Comment:
Just thought of something, please feel free to ignore if you disagree.

I'm assuming the slider will have to be accessible, support swiping, keyboard arrows etc.

In that case, we might consider changing the `handleClick` to something more applicable in various event scenarios like `handleSelect` or `handleTransition`.

Reply:
Mmm, yeah good point! I think that we should improve this when the slider, next, and previous buttons and animation are done!

What do you think?

Reply:
Sounds good to me!

Reply:
You are right! https://github.com/woocommerce/woocommerce-blocks/pull/10823/files#r1325598086!